backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-backing-up-pvs-csi-doc.adoc

```diff
@@ -34,8 +34,8 @@ driver: <csi_driver>
 deletionPolicy: <deletion_policy_type> <3>
 ----
 <1> Must be set to `true`.
-<2> Must be set to `true`.
-<3> OADP supports the `Retain` and `Delete` deletion policy types for CSI and Data Mover backup and restore. For the OADP 1.2 Data Mover, set the deletion policy type to `Retain`.
+<2> Optional: Must be set to `true`. If you are restoring this volume in another cluster with the same driver, make sure that you set the `snapshot.storage.kubernetes.io/is-default-class` parameter to `false` instead of setting it to `true`. Otherwise, the restore will partially fail.
```

Comment:
I think it is confusing to say optional and that it must be true. Would be better to remove that part and just have the `If you are restoring this volume ...` part?

Reply:
Done, thanks